etcdctl/ctlv2/ctl.go

```diff
@@ -25,15 +25,23 @@ import (
 	"github.com/urfave/cli"
 )
 
-func Start() {
+func Start(apiv string) {
 	app := cli.NewApp()
 	app.Name = "etcdctl"
 	app.Version = version.Version
 	cli.VersionPrinter = func(c *cli.Context) {
 		fmt.Fprintf(c.App.Writer, "etcdctl version: %v\n", c.App.Version)
 		fmt.Fprintln(c.App.Writer, "API version: 2")
 	}
-	app.Usage = "A simple command line client for etcd."
+	app.Usage = "A simple command line client for etcd.\n"
```

Comment:
also, this shouldn't have an extra new line since it changes the output to have an extra newline even if the env var is set? The extra newline should probably be `app.Usage += "\n\n"` instead of 'app.Usage += "\n"`. Right now it gives:
```
NAME:
   etcdctl - A simple command line client for etcd.


USAGE:
   etcdctl [global options] command [command options] [arguments...]
```

instead of 

```sh
NAME:
   etcdctl - A simple command line client for etcd.

USAGE:
   etcdctl_test [global options] command [command options] [arguments...]
```